presto-hive/src/main/java/com/facebook/presto/hive/RecordFileWriter.java

```diff
@@ -122,6 +125,15 @@ public long getWrittenBytes()
             return ((ExtendedRecordWriter) recordWriter).getWrittenBytes();
         }
 
+        if (committed) {
+            try {
+                return path.getFileSystem(conf).getFileStatus(path).getLen();
+            }
+            catch (IOException e) {
+                throw new UncheckedIOException(e);
+            }
+        }
+
         // there is no good way to get this from RecordWriter
```

Comment:
The comment is now somewhat misleading. Please update.

Reply:
Don't know how to rephrase it without being oververbose. Any help is appreciated.